featuretools/entityset/entityset.py

```diff
@@ -1113,7 +1117,7 @@ def add_last_time_indexes(self, updated_dataframes=None):
                         name=df.ww.name,
                         index=df.ww.index,
                         time_index=df.ww.time_index,
-                        logical_types=df.ww.logical_types,
+                        logical_types={**df.ww.logical_types, LTI_COLUMN_NAME: lti_ltype},
```

Comment:
Ran into a small bug when trying call `add_last_time_indexes` on an empty DataFrame where we couldn't do inference. Definitely an edge case, but I figured it couldn't hurt to avoid type inference here.